payment-request/payment-request-show-method.https.html

```diff
@@ -54,9 +46,9 @@
   await promise_rejects_dom(t, "SecurityError", request.show());
 
   await test_driver.bless();
-  const acceptPromise = request.show(); // Sets state to "interactive"
-
-  await promise_rejects_dom(t, "InvalidStateError", request.show());
+  // Sets state to "interactive", but consumes user interaction.
+  const acceptPromise = request.show();
+  await promise_rejects_dom(t, "SecurityError", request.show());
```

Comment:
Dumb question: If the `test_driver.bless()` has been run, then why is `request.show()` rejecting with `"SecurityError"`?

Reply:
bless() starts the transient activation. 

The first .show() after bless() “consumes the transient activation”.

so when show() is called again, a SecurityError gets thrown because the transient activation has been consumed.